addons/web/static/src/core/code_editor/code_editor.js

```diff
@@ -169,6 +172,19 @@ export class CodeEditor extends Component {
             () => [this.props.sessionId, this.props.mode, this.props.value]
         );
 
+        useEffect(
```

Comment:
The codeEditor has been thought of as context-agnostic: it should not care to be in the context of the relational model, and should not receive record specifically.
There are a few solution:
- Build an augmented CodeEditor extending that one that the AceField would use
- Define a/some props that the Code editor just execute or use, and pass those props from the aceField